src/main/java/ejercicio/banco/repository/MySqlAccountRepository.java

```diff
@@ -2,22 +2,97 @@
 
 import ejercicio.banco.dto.Account;
 
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.util.ArrayList;
 import java.util.List;
 
+import static ejercicio.banco.util.DataBaseUtil.closeConections;
+import static ejercicio.banco.util.DataBaseUtil.startConnection;
+
 public class MySqlAccountRepository implements AccountRepository {
+
+    private static final String FILENAME = "bank_project";
+
     @Override
     public List<Account> findAll(String filename) {
-        return null;
+        List<Account> accounts = new ArrayList();
+
+        Connection con = startConnection(filename);
```

Comment:
You should be using here the `DataBaseUtil.startConnection()`right?

Reply:
Done on last commit